packages/gamut/src/Form/styles/shared-system-props.tsx

```diff
@@ -89,3 +97,113 @@ export const conditionalStyles = system.variant({
 export const conditionalStyleState = (error: boolean, activated: boolean) => {
   return error ? 'error' : activated ? 'activated' : undefined;
 };
+
+export const radioWrapper = system.css({
+  margin: '0.25rem 0',
+  width: '100%',
+  fontWeight: 'normal',
+});
+
+const consistentLabelStyles = {
+  content: '""',
+  display: 'block',
+  width: 20,
+  height: 20,
+  borderRadius: '100%',
+  mr: 8,
+} as const;
+
+export const radioLabel = system.css({
+  display: 'flex',
+  padding: '1rem 0',
+  alignItems: 'center',
+  cursor: 'pointer',
+  position: 'relative',
+  '&::before': {
+    ...consistentLabelStyles,
+    bg: 'background',
+    boxShadow: `0 0 0 1px ${theme.colors.primary}`,
+    transition: timing.slow,
+  },
+  '&::after': {
+    ...consistentLabelStyles,
+    position: 'absolute',
+    transition: `transform ${timing.slow} ease-in-out`,
```

Comment:
Since we're already importing it here :)
```suggestion
    transition: transitionConcat(['transform'], 'slow', 'ease-in-out'),
```